app/src/main/java/org/mixdrinks/mixdrinks/app/MixDrinksApp.kt

```diff
@@ -34,13 +41,37 @@ fun MixDrinksApp(modifier: Modifier = Modifier) {
                     DetailScreen(
                         modifier = modifier,
                         cocktailId = it,
+                        onNavigateToDetailGood = { navController.navigate("good/$it")},
+                        onNavigateToDetailTool = { navController.navigate("tool/$it")},
+                        onBack = { navController.popBackStack() }
+                    )
+                }
+            }
+            composable("good/{goodId}") {
+                    backStackEntry ->
+                val cocktailId = backStackEntry.arguments?.getString("goodId")
```

Comment:
Move "goodId" string to some const val and use the constants insted of hardcode string